lib/inventory_refresh/inventory_collection/builder.rb

```diff
@@ -155,6 +174,12 @@ def to_hash
 
       protected
 
+      def assert_allowed_property(name)
+        unless allowed_properties.include?(name)
+          raise "InventoryCollection property :#{name} is not allowed. Allowed properties are:\n#{self.allowed_properties.to_a.map(&:to_s).join(', ')}"
```

Comment:
the self. is really redundant, you've probably missed that in the rubocop message